### ISSUE TITLE
训练异常中断，弹出“Python已停止工作”的窗口

### ISSUE THREAD

**Issue by Caoxiaocao** [tanluren/yolov3-channel-and-layer-pruning#28]
Reading labels (8952 found, 0 missing, 0 empty for 8967 images): 100%|███████████▉| 8952/8967 [00:37<00:00, 
Reading labels (8967 found, 0 missing, 0 empty for 8967 images): 100%|████████████| 8967/8967 [00:37<00:00, 237.71it/s]
Model Summary: 222 layers, 6.1626e+07 parameters, 6.1626e+07 gradients
Starting training for 273 epochs...

     Epoch   gpu_mem      GIoU       obj       cls     total      soft   targets  img_size
  0%|                                                                                          | 0/561 [00:00<?, ?it/s]learning rate: 1e-06

(Pytorch) F:\DeepLearning\YOLO优化\yolov3-channel-and-layer-pruning-master>    

你好，我训练开始不久程序就崩溃了，然后弹出了“Python已停止工作”的窗口。请问这是什么原因呢，该如何解决？
